pom.xml

```diff
@@ -45,7 +45,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.5.4</version>
+            <version>[2.9.9.1,)</version>
```

Comment:
@bethleha Brackets and comma in version?

Reply:
@anithakathir This is a version range specification which can be translated as `2.9.9.1 or later`. Please refer this https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html for more info.